src/plugins/azgenerator/AzCoreIncrementalGenerator.ts

```diff
@@ -63,6 +64,10 @@ export class AzCoreIncrementalGenerator extends AzGeneratorBase {
         // Add Import from generated folder (Help)
         await this.generateIncrementalSingleAndAddtoOutput(new CliTopHelp(this.model, this.isDebugMode));
 
+        // Add Import from generated folder (Report)
+        const AzureCliReportGenerator = new CliReport(this.model, this.isDebugMode);
+        this.files[path.join(this.model.azOutputFolder, "report.md")] = await AzureCliReportGenerator.incrementalGeneration("");
```

Comment:
await this.generateIncrementalSingleAndAddtoOutput(new CliReport(this.model, this.isDebugMode));